run/README.namelist

```diff
@@ -535,16 +544,17 @@ Namelist variables for controlling the adaptive time step option:
                                                  (only used in very extreme situation)
                                                  default value = 0
                                                  gsfcgce_hail is ignored if gsfcgce_2ice is set to 1 or 2.
- hail_opt                            = 0       ! hail switch for WSM6 and WDM6 : 0 - off, 1 - on (new in 3.6.1)
+ hail_opt                            = 0       ! hail switch for WSM6 and WDM6 : 0 - off, 1 - on
  morr_rimed_ice                      = 1       ! hail switch for Morrison schemes (mp_physics=10 and 40: 0 - off, 1 - on (new in 4.0)
  clean_atm_diag                      = 0       ! if set to =1, turns on clean sky diagnostics (for chem); default is 0=off
+ acc_phy_tend                        = 1       ! option to output 16 accumulated physics tendencies for potential temp, water vaopr mixing ratio, and U/V wind components; default is 0=off (new in 4.4) 
  progn (max_dom)                     = 0       ! switch to use mix-activate scheme (Only for Morrison, WDM6, WDM5, 
                                                  and NSSL_2MOMCCN/NSSL_2MOM
- ccn_conc                            = 1.E8    ! CCN concentration, used by WDM schemes (new in 3.6.1)
+ ccn_conc                            = 1.E8    ! CCN concentration, used by WDM schemes
 
  no_mp_heating                       = 0       ! normal
                                      = 1       ! turn off latent heating from a microphysics scheme
- use_mp_re                           = 1       ! whether to use effective radii computed in mp schemes in RRTMG (new in 3.8).
+ use_mp_re                           = 1       ! whether to use effective radii computed in mp schemes in RRTMG
                                                  0: do not use; 1: use effective radii
                                                  (The mp schemes that compute effective radii are 3,4,6,8,14,16,17-21,28)
```

Comment:
@kkeene44 Since you're editing this file, can you also add more mp schemes that use use_mp_re? The options to add are 7, 10, 24, 26, 50-53, 55.
More options should be added for do_radar_ref too: 24, 26.

Reply:
@weiwangncar Yes, I'll add those. Do you want me to go through and compare this file to the registry to make sure we have everything in there, and that no older items need to be removed?

Reply:
@kkeene44 Only if it doesn't take too much time, or if you have some easy way to do so.

Reply:
@weiwangncar Yeah, I think that may be a bigger project. I'll hold off on that for now and may come back to it later. For now, I've updated the 2 options you mentioned.